README.md

```diff
@@ -57,32 +57,8 @@ conda install -c bioconda snakefmt
 
 ### Containers
 
-#### Docker
-
-[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/snakemake/snakefmt)](https://hub.docker.com/r/snakemake/snakefmt/tags)
-
-```sh
-docker pull snakemake/snakefmt
-docker run -it snakemake/snakefmt snakefmt --help
```

Comment:
I like the idea of using bioconda's images- however I believe we should still give an example of how to run it, as here